plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergUtil.java

```diff
@@ -332,4 +345,50 @@ public static LocationProvider getLocationProvider(SchemaTableName schemaTableNa
         }
         return locationsFor(tableLocation, storageProperties);
     }
+
+    public static Schema toIcebergSchema(List<ColumnMetadata> columns)
+    {
+        List<Types.NestedField> icebergColumns = new ArrayList<>();
+        for (ColumnMetadata column : columns) {
+            if (!column.isHidden()) {
+                int index = icebergColumns.size();
+                org.apache.iceberg.types.Type type = toIcebergType(column.getType());
+                Types.NestedField field = column.isNullable()
```

Comment:
I think this can use the `of()` method:
```java
NestedField field = NestedField.of(index, column.isNullable(), column.getName(), type, column.getComment());
```
Note that you can import `NestedField` directly so it doesn't need qualification